src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -155,6 +160,67 @@ private void OnGUI()
             // project = EditorGUILayout.TextField("Project", project);
             // EditorGUILayout.EndToggleGroup();
         }
+
+        /// <summary>
+        /// Creates Sentry folder for storing its configs - Assets/Resources/Sentry
+        /// </summary>
+        private static void TryCreateSentryFolder()
+        {
+            // TODO: revise, 'Resources' is a special Unity folder which is created by default. Not sure this check is needed.
+            if (!AssetDatabase.IsValidFolder("Assets/Resources"))
+            {
+                AssetDatabase.CreateFolder("Assets", "Resources");
+            }
+            if (!AssetDatabase.IsValidFolder($"Assets/Resources/{UnitySentryOptions.ConfigRootFolder}"))
+            {
+                AssetDatabase.CreateFolder("Assets/Resources", UnitySentryOptions.ConfigRootFolder);
+            }
+        }
+
+        /// <summary>
+        /// Find and copy 'link.xml' into current Unity project for IL2CPP builds
+        /// </summary>
+        private static void TryCopyLinkXml()
+        {
+            const string linkXmlFileName = "link.xml";
+
+            var linkXmlPath = $"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/{linkXmlFileName}";
+            if (File.Exists(linkXmlPath))
+            {
+                return;
+            }
+
+            Debug.Log($"'{linkXmlFileName}' is not found. Creating one!");
+
+            var linkPath = GetLinkXmlPath(linkXmlFileName);
+            if (linkPath == null)
+            {
+                Debug.LogWarning($"Couldn't locate '{linkXmlFileName}' in 'Packages'.");
+                return;
+            }
+
+            var linkXmlAsset = AssetDatabase.LoadAssetAtPath<TextAsset>(linkPath);
+            File.WriteAllBytes(linkXmlPath, linkXmlAsset.bytes);
+        }
+
+        /// <summary>
+        /// Get Unity path to 'link.xml' file from `Packages` folder
+        /// </summary>
+        private static string? GetLinkXmlPath(string linkXmlFileName)
+        {
+            var assetIds = AssetDatabase.FindAssets("io.sentry.unity", new [] { "Packages" });

```

Comment:
Best to lift this `io.sentry.unity` elsewhere so we can follow the reference and document what this means and effects of changing it.
```suggestion
            var assetIds = AssetDatabase.FindAssets("io.sentry.unity", new [] { "Packages" });

```